src/Zicht/Bundle/FrameworkExtraBundle/Resources/config/services.xml

```diff
@@ -43,5 +48,10 @@
         <service id="Zicht\Bundle\FrameworkExtraBundle\Url\UrlCheckerService" class="Zicht\Bundle\FrameworkExtraBundle\Url\UrlCheckerService">
             <argument type="service" id="request_stack"/>
         </service>
+
+        <service id="Zicht\Bundle\FrameworkExtraBundle\JsonSchema\SchemaService" class="Zicht\Bundle\FrameworkExtraBundle\JsonSchema\SchemaService">
+            <argument type="service" id="translator"/>
+            <argument>%kernel.root_dir%/../web</argument>
```

Comment:
Mogelijk verbetering die ook forward-compatible is zou het gebruiken van `kernel.project_dir` zijn, dit kan al vanaf SF3.3. https://symfony.com/doc/3.4/reference/configuration/kernel.html#project-directory `kernel.root_dir` is deprecated vanaf 4.x.

Reply:
Goed idee!.  Aangepast naar `%kernel.project_dir%/web`